functests/vrf/vrf.go

```diff
@@ -244,3 +188,63 @@ func paramInParamList(param string, paramRange []string) error {
 	}
 	return fmt.Errorf("error: wrong parameter %v", param)
 }
+
+func defineDesсribeTable(node string, ipStack string, namespace string, apiclient *client.ClientSet, nodeName string, testNetworkBlue string, testNetworkRed string) {
+	var podClientVRFBlueIPAddress string
+	var podServerVRFBlueIPAddress string
+
+	By("Create overlapping IP pods")
+	podClientVRFRedOverlappingIP := getOverlapIP(apiclient, nodeName, "overlap-client-ip", namespace)
+	podServerVRFRedOverlappingIP := getOverlapIP(apiclient, nodeName, "overlap-server-ip", namespace)
+
+	if ipStack == ipStackIPv4 && net.ParseIP(podClientVRFRedOverlappingIP).To4() == nil {
+		Skip("Skipping IPv4 test. Cluster supports IPv6 protocol")
+	} else if ipStack == ipStackIPv4 && net.ParseIP(podClientVRFRedOverlappingIP).To4() != nil {
+		podClientVRFBlueIPAddress = "10.255.255.1"
+		podServerVRFBlueIPAddress = "10.255.255.2"
+	} else {
+		Skip("Unpsupported protocol parameter")
+	}
+
+	By("Create VRFs client/server pods")
+	podClientIpamConfig := fmt.Sprintf(`[{"name": "%s", "mac": "%s", "ips": ["%s/24"]}, {"name": "%s", "mac": "%s", "ips": ["%s/24"]}]`,
```

Comment:
this is more than the ipam config, I Think this is the podNetworkDefinition. You can omit pod and just call it clientNetworkDefinition